int64 id

int64 from_id
int64 to_id

geometry_msgs/Pose relative_pose

# Row-major representation of the 6x6 information matrix
float64[36] information